the-collab-lab/tcl-37-smart-shopping-list
# Create Button Component (Orange Button)

## Description
In this PR, I added a Buttons folder to the components directory, and created the OrangeButton component. ( components > Buttons > OrangeButton )
This is a reusable button that is passed text and onClick functions, making it customizable for "Create new List", "Join Existing List", and "Add Item". 
![Screen Shot 2022-02-25 at 2 50 58 PM](https://user-images.githubusercontent.com/67841237/155785681-f830699c-8be8-45e5-818b-83a8ac057373.png)

## Related Issue
Closes #40 



## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
| ✓   | :sparkles: New feature     |



## Updates

### Before
![Screen Shot 2022-02-26 at 10 34 15 AM](https://user-images.githubusercontent.com/67841237/155849010-49118305-a5f0-4df9-8fb8-5bf67fadf00b.png)



### After
![Screen Shot 2022-02-26 at 10 35 04 AM](https://user-images.githubusercontent.com/67841237/155849024-154e6891-b647-4127-b9af-d00faf3d4b72.png)
![Screen Shot 2022-02-26 at 10 35 41 AM](https://user-images.githubusercontent.com/67841237/155849053-d6d61496-4390-4dab-a6f3-e8a462fb0aa2.png)



## Testing Steps / QA Criteria


1. The buttons appear on the Home page, and Add Item view. 
2. You can click the corresponding button to test functionality. (Short video provided)

https://user-images.githubusercontent.com/67841237/155849279-8c1de6b3-5823-4c2c-85d0-fb6ffb4bab1d.mp4
